Lib/ufo2ft/__init__.py

```diff
@@ -54,6 +54,7 @@ def compileOTF(
     subroutinizer=None,
     notdefGlyph=None,
     _tables=None,
+    _reloadFont=True,
```

Comment:
How about we add a `postProcessorClass` similar to `outlineCompilerClass`, `preProcessorClass` that already exist, only that in the case of postProcessorClass it is `Optional[Type]` so caller can set it to `None` and thus have compile skip the post-process. I think it'd be clearer that way. Using a `_reloadFont` option makes the assumption that post-processing by necessity entains a reloading.